README.rst

```diff
@@ -150,6 +173,74 @@ Note: gcm_send_bulk_message must be used when sending messages to topic subscrib
 
 Reference: `GCM Documentation <https://developers.google.com/cloud-messaging/topic-messaging>`_
 
+Multiple mobile applications
+----------------------------
+
+In order to use multiple mobile applications on the same server, you should provide some method to associate the application IDs
+and correspondent application keys or certificates.
+
+If your are planning to use some not big and static number of mobile applications, you can use a static dictionary directly in
+your settings file. Use ``APNS_CERTIFICATES`` and ``GCM_API_KEYS`` settings to store this dictionary for APNS and GCM
+correspondently. The ``APNS_CERTIFICATE`` and ``GCM_API_KEY`` settings are used as defaults.
+
+.. code-block:: python
+
+	PUSH_NOTIFICATIONS_SETTINGS = {
+		"GCM_API_KEY": "<your default application api key>",
+		"GCM_API_KEYS": {
+		    "<application ID 1>":"<your api key 1>",
+		    "<application ID 2>":"<your api key 2>",
+		    ...
+		},
+		"APNS_CERTIFICATE": "/path/to/your/default/certificate.pem",
+		"APNS_CERTIFICATES": {
+		    "<application ID 1>":"/path/to/your/certificate1.pem",
+		    "<application ID 2>":"/path/to/your/certificate2.pem",
+		    ...
+		}
+	}
+
+If your are planning to use dynamic and/or big number of mobile applications, you can use a dynamic access to
+the database table containig records describing applications. Use ``APNS_CERTIFICATES_MODEL``
+and ``GCM_API_KEYS_MODEL`` settings to describe model(s) containing reflection
+of application IDs to the correspondent values. The ``APNS_CERTIFICATE`` and ``GCM_API_KEY`` settings are used
+as defaults.
+
+Let say, the application ``applications`` contains a model ``ApplicationModel`` which contains three fields to
+reflect an application ID to the correspondent application credentials:
+
+.. code-block:: python
+
+    class ApplicationModel(models.Model):
+        application_id = models.CharField(max_length=64,unique=True)
+        gcm_api_key = models.TextField(null=True,blank=True)
+        apns_certificate = models.FileField(null=True,blank=True)
```

Comment:
As I mentioned in the previous PR, using a FileField doesn't work for horizontal scaling of servers, as the cert file would be uploaded onto a single server.

I also realize that your `ApplicationModel` here is a sample model definition - can you elaborate how the supporting code in this PR would allow the storage of the certfile contents in the DB to allow for horizontal scaling? 

In my previous #339 PR, I had some code that temporarily wrote the content out to a file, then remove it after the call succeeded.  Not efficient, but could something like this be supported? Otherwise, I'm not sure APNS in a model makes much sense. Thanks!